drivers/timer/Kconfig.omap_dm_timer

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config OMAP_DM_TIMER
+	bool "TI Dual-Mode Timer"
+	default y
+	select TICKLESS_CAPABLE
+	depends on DT_HAS_TI_AM654_DMTIMER_ENABLED
+	help
+		This module implements a kernel device driver for TI dual-mode timer. This
```

Comment:
Kconfig help indent is 1x tab followed by 2x spaces

Reply:
Fixed